clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -180,6 +189,15 @@ def visits_to_timepoints_mrilist(subject, mri_list_subj, adnimerge_subj, modalit
 
 
 def select_image_qc(id_list, mri_qc_subj):
+    """
+
```

Comment:
Provide description of function, with details on the different arguments

Reply:
Done